controllers/connectionTypeController.js

```diff
@@ -95,6 +97,8 @@ exports.updateConnectionType = async (req, res, next) => {
 		await connectionTypeService.patchConnectionType(id, updateDetail)
 		res.status(200).json({ status: "success", message: `Connection updated successfully.` })
 	} catch (error) {
-		next(error)
+		if (error.code !== 11000) next(error)
```

Comment:
if (error.code !== 11000)
this check should not be in the controller but in the service function instead.

its fine if you add try catch there